tests/e2e/hcp_cluster_test.go

```diff
@@ -0,0 +1,189 @@
+				Expect(err).To(BeNil())
+				Expect(output.String()).To(ContainSubstring("--kms-key-arn"))
+				Expect(output.String()).To(ContainSubstring("--etcd-encryption"))
+				Expect(output.String()).To(ContainSubstring("--enable-customer-managed-key"))
+
+				By("Get cluster description")
+				jsonData, err := clusterService.GetJSONClusterDescription(clusterID)
+				Expect(err).To(BeNil())
+
+				if !isHosted {
+					Skip("It doesn't support on non HCP cluster - cannot run this test")
+				}
+
+				ectdKMS := jsonData.DigString("aws", "etcd_encryption", "kms_key_arn")
+				npKMS := jsonData.DigString("aws", "kms_key_arn")
+
+				if clusterConfig.EtcdEncryption {
+					Expect(clusterConfig.Encryption.EtcdEncryptionKmsArn).To(Equal(ectdKMS))
+				}
+				if clusterConfig.EnableCustomerManagedKey {
+					Expect(clusterConfig.Encryption.KmsKeyArn).To(Equal(npKMS))
+				}
+
+			})
+
+		It("create HCP cluster with network type can work well via rosa cli - [id:71050]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Check the help message of 'rosa create cluster -h'")
+				//It is hiddened now
+				helpOutput, err, _ := clusterService.Create("", "-h")
+				Expect(err).To(BeNil())
+				Expect(helpOutput.String()).ToNot(ContainSubstring("--no-cni"))
+
+				By("Get cluster description")
+				if !isHosted {
+					Skip("It does't non HCP cluster - cannot run this test")
+				}
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+				networkLine := clusterDetail.Network[0]
+
+				By("Get cluster description via json")
+				jsonData, err := clusterService.GetJSONClusterDescription(clusterID)
+				Expect(err).To(BeNil())
+				Expect(networkLine["Type"]).To(Equal(jsonData.DigString("network", "type")))
+				if clusterConfig.Networking != nil {
+					networkType := clusterConfig.Networking.Type
+					if networkType != "" && networkType == "Other" {
+						Expect(networkLine["Type"]).To(Equal("Other"))
+					}
+				}
+			})
+
+		It("create ROSA HCP cluster with external_auth_config config should work well via rosa client - [id:71945]",
+			labels.High, labels.NonClassicCluster,
+			func() {
+				By("Check the help message of 'rosa create cluster -h'")
+				helpOutput, err, _ := clusterService.Create("", "-h")
+				Expect(err).To(BeNil())
+				Expect(helpOutput.String()).To(ContainSubstring("--external-auth-providers-enabled"))
+
+				By("Check if cluster enable external_auth_config")
+				if !isHosted {
+					Skip("It does't non HCP cluster - cannot run this test")
+				}
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+
+				if !clusterConfig.ExternalAuthentication {
+					Skip("It is only for external_auth_config enabled clusters")
+				}
```

Comment:
`Skip` will totally skip the result in the Junit.xml. If there are some checkpoints before `Skip` are available, please use `return` instead.